docs/how-to/nat-configuration.md

```diff
@@ -1,32 +1,132 @@
+### Use DCUtR Holepunching
+
+As of Kubo v0.13, [DCUtR hole punching is enabled by default](https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.13.md#-relay-v2-client-with-auto-discovery-swarmrelayclient).
+
+DCUtR holepunching has various drawbacks and tradeoffs. Currently, the connection signaling goes through a relay, which adds an average latency of 5 seconds when opening a connection. Once a direct connection is established, latency is normal. Additionally, DCUtR holepunching does not have a 100% success rate and can fail. For a deeper dive into how holepunching works in Kubo, it's drawbacks, and more information on using DCUtR, see the [_Hole punching in libp2p - Overcoming Firewalls_ blog post](https://blog.ipfs.tech/2022-01-20-libp2p-hole-punching/). 
+
+Because of these drawbacks, you may want to use another solution, like manual port forwarding. To enable manual port forwarding, see the instructions below.
+
+### Enable manual port forwarding
+
+If your router does not support UPNP and/or IPv6, or you want better reliability and performance than what DCUtR provides, set up manual port forwarding. Complete the following steps to enable manual port forwarding:
+
+1. [Open a port from the internet to your internal Kubo node](#open-a-port). 
+1. [Update the Kubo Configuration](#update-the-kubo-configuration).
+1. [Restart your Kubo node](#restart-your-kubo-node). 
+
+First, open a port from the internet to your internal Kubo node.
+
+#### Open a port 
+
+Each router has different options and solutions for port forwarding. Most router manufacturers have guides for setting up port forwarding on their devices. In general, the steps are:
 
 1. Log into your router.
 1. Locate your routers port forwarding section.
 1. Enter the IP address of your IPFS node.
-1. Set traffic to go through port `4001`.
+1. Set traffic to go through an outside port. 
+   :::tip
+   If you are unsure of what port to use, `4001` is recommended.
+   :::
 1. Reboot your IPFS node for the changes to take effect. Make sure to reboot the entire machine, not just the IPFS daemon.
 
-Assuming you're not trying to expose your daemon's API to the public internet, opening port `4001`/`tcp` should be sufficient.
+Now that you've opened a port from the internet to your internal Kubo node, update your Kubo configuration to set `Swarm.AppendAnnounce` as a list of addresses that other IPFS nodes will try to contact you at.
+
+#### Update the Kubo Configuration
+
+In this step, you will update your Kubo configuration to set `Swarm.AppendAnnounce` as a list of addresses that other IPFS nodes will try to contact you at. This list lets other nodes on the network that the port forward you created in the previous step know that your node exists, and that these are the addresses at which you can be contacted at. To update your configuration:
```

Comment:
Something went wonky with this paragraph.

"In this step, you will update your Kubo configuration to set `Swarm.AppendAnnounce` as a list of addresses that other IPFS nodes will try to contact you at. This list lets other nodes on the network (**know**) that the port forward you created in the previous step ~~know that your node~~ exists, and that these are the addresses at which you can be contacted at.

Something like that.

Reply:
Your rewrite makes sense, I vote yes